src/layers/layer-update.js

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable guard-for-in */
+
+/**
+ *
+ * @param {Object} updateTriggers {getPosition: {column}, getData: {filteredIndex}}
+ * @param {Object} oldUpdateTriggers
+ * @returns {}
+ */
+export function diffUpdateTriggers(updateTriggers, oldUpdateTriggers = {}) {
+  const triggerChanged = {};
+  let reason = false;
+
+  for (const triggerName in updateTriggers) {
```

Comment:
you can rewrite this loop using Object.entries(object) 
```
Object.entries(updateTriggers).reduce((acc, pair) => {
  const {[0]: triggerName, [1]: newTriggers = {}} = pair;
  const oldTriggers = oldUpdateTriggers[triggerName] || {};
  const diffReason = compareUpdateTrigger(newTriggers, oldTriggers, triggerName);
 .....
}, {triggerChanged});
```

Reply:
The argument here is shorter code or code that's easier to understand. I feel like reduce is fancier, but in a function like this I feel like readability is more important